lib/crypto.js

```diff
@@ -229,6 +229,8 @@ ObjectDefineProperty(constants, 'defaultCipherList', {
   value: getOptionValue('--tls-cipher-list')
 });
 
+let promises;
```

Comment:
In current `master`, the `promises` in `fs` are defined as:

```js
let promises = null;
```

then below in the `get()` method:

```js
if (promises === null) {
// ..
}
```